include/seqan/arg_parse/argument_parser.h

```diff
@@ -237,7 +237,7 @@ class ArgumentParser
                                         ArgParseArgument::STRING,
                                         "OPTION"));
         setValidValues(*this, "version-check", "DEV OFF APP_ONLY");
-        setDefaultValue(*this, "version-check", VersionControlTags::OPTION_DEV);
+        setDefaultValue(*this, "version-check", VersionControlTags_<>::OPTION_DEV);
```

Comment:
Maybe we should deactivate it here, instead of through the macro? Would make it easier to experiment with.